plugins.txt

```diff
@@ -138,9 +138,9 @@ ssh-agent:333.v878b_53c89511
 ssh-credentials:305.v8f4381501156
 sshd:3.275.v9e17c10f2571
 structs:324.va_f5d6774f3a_d
-support-core:1274.v097a_073e7733
+support-core:1278.vf26a_78c372f1
 theme-manager:193.vcef22f6c5f2b_
-timestamper:1.24
+timestamper:1.25
```

Comment:
Cleanup of unused dependencies: https://github.com/jenkinsci/timestamper-plugin/releases/tag/timestamper-1.25